_includes/releases/v23.1/v23.1.0-alpha.3.md

```diff
@@ -0,0 +1,102 @@
+- Fixed a bug that caused the server to crash if trying to restore a table from a backup generated by `BACKUP TABLE` from a schema that includes [user-defined functions](../v22.2/user-defined-functions.html), and the restore target database does not have a schema with the same name. [#96911][#96911] 
+- Changefeeds with the `unordered` option can use multi-region [Google Cloud Pub/Sub topics](../v22.2/changefeed-sinks.html). [#96567][#96567] 
+- Fixed a bug in changefeeds where long running [initial scans](../v22.2/create-changefeed.html#initial-scan) will fail to generate checkpoints. Failure to generate [checkpoints](../v22.2/change-data-capture-overview.html#how-does-an-enterprise-changefeed-work) is particularly bad if the changefeed restarts. Without checkpoints, the changefeed will restart from the beginning, and in the worst case, when exporting substantially sized tables, the changefeed initial scan may not complete. [#96995][#96995]
+
+<h3 id="v23-1-0-alpha-3-sql-language-changes">SQL language changes</h3>
+
+- Added [latency information](https://www.cockroachlabs.com/docs/cockroachcloud/statements-page.html#statement-statistics) in seconds to the statement statistics on `crdb_internal.statement_statistics`, `system.statement_statistics`, and `crdb_internal.cluster_statement_statistics`, with information about: `min`, `max`, `p50`, `p90`, and `p99. Also added the columns: `latency_seconds_min`, `latency_seconds_max`, `latency_seconds_p50`, `latency_seconds_p90`, and `latency_seconds_p99` to `crdb_internal.node_statement_statistics`.[#96396][#96396]
+- Deprecated the PGDUMP and MYSQLDUMP formats for [IMPORT](../v22.2/import-performance-best-practices.html). They are still present, but will be removed in a future release. View this [page](https://www.cockroachlabs.com/docs/v22.2/migration-overview) for alternatives. [#96386][#96386]
+- [`COPY ... FROM ... QUOTE '"'`](https://www.cockroachlabs.com/docs/v22.2/copy-from) will no longer error. [#96572][#96572]
+- Added `last_error_code` column to the `crdb_internal.node_statement_statistics` table. Added `last_error_code` field to the `statistics` JSON blob in the `crdb_internal.statement_statistics` and `system.statement_statistics` tables. [#96436][#96436]
+- Added support for expressions of the form `COLLATE "default"`, `COLLATE "C"`, and `COLLATE "POSIX"`. Since the default [collation](../v22.2/collate.html) cannot be changed currently, these expressions are all equivalent. The expressions are evaluated by treating the input as a normal string, and ignoring the collation.  This means that comparisons between strings and collated strings that use "default", "C", or "POSIX" are now supported.  Creating a column with the "C" or "POSIX" collations is still not supported. [#96828][#96828]
```

Comment:
```suggestion
- Added support for expressions of the form `COLLATE "default"`, `COLLATE "C"`, and `COLLATE "POSIX"`. Since the default [collation](../v22.2/collate.html) cannot be changed currently, these expressions are all equivalent. The expressions are evaluated by treating the input as a normal string, and ignoring the collation.  This means that comparisons between strings and collated strings that use `"default"`, `"C"`, or `"POSIX"` are now supported.  Creating a column with the `"C"` or `"POSIX"` collations is still not supported. [#96828][#96828]
```